packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -87,7 +97,7 @@ async function generateBlogFeed({
       description,
       // Atom feed demands the "term", while other feeds use "name"
       category: tags.map((tag) => ({name: tag.label, term: tag.label})),
-      content: mdxToFeedContent(post.content),
+      content: $('#post-content').html()!,
```

Comment:
Maybe we'll need to extract a default constant value in a shared package? 

docusaurus-utils-common is shared between node+browser, looks like a good fit

Eventually, add a new config option to avoid tight coupling between theme/plugin? (probably useless, we can do that later)